### ISSUE TITLE
Bug Report (Dev Build 411)

### ISSUE THREAD

**Issue by Mustang-25** [dcs-liberation/dcs_liberation#386]
Gave the latest dev build (Build 411 at the time) a test and I have some bug reports and observations:

1. Toggling culling zones on will hide all the units on the liberation map and just display a blank map

2. Ships no longer display their SAM ranges even when all the other SAM ranges are displayed

3. An empty ship objective (CAIMAN) is generated

4. A ship objective is generated on land (PYTHON) in the Liberation map but it is properly placed on water in the ME

5. Egress points for the planned anti-ship strikes take them directly over enemy SAM sites

6. The F-15C can be fragged for an anti-ship strike in Liberation but it will default to CAS in the ME (presumably other aircraft that are pure A2A will have this same problem)

7. Single flight anti-ship strikes still get generated with join and split points

**Comment by walterroach** [comment]
[MissionAndSave.zip](https://github.com/Khopa/dcs_liberation/files/5555338/MissionAndSave.zip)

**Comment by walterroach** [comment]
4 is the known map projection issue here: #91 , see also pydcs/dcs#38

**Comment by Mustang-25** [comment]
So I actually ran the mission and here are my observations from it: 

There was one flight each Hornet, Jeff, and Viggen to see what each would do.

The Hornets each fired 3 out of 4 Harpoons before bugging out. No idea why they didn't fire their 4th one.

The Viggen each fired 1 of 2 Rb-15s and waited until their first Rb-15 had hit before firing the second one. This led them to be much closer to the ships than the waypoints intended.

The Jeffs didn't fire any of their 802-AKs, they never made it to their ingress point before turning around and trying to hit the tanker. I noticed they were using a lot of burner on their way to the ingress point but not sure why (possibly because the airfield they were taking off from was super congested so they were trying to get to their fragged TOT). A stop gap solution might be replacing the TGP on the centerline with a fuel tank.
